### PR TITLE
Added css class names "today" and "tomorrow" for calendar

### PR DESCRIPTION
**Pull request by retroflex**
Added class names "today" and "tomorrow" on the calendar module tr elements (i.e. calendar items).
This way you can for example color your events today and/or tomorrow to more easily see what's happening in the near future.

Implemented by adding an event.tomorrow variable (similar to event.today) that can be used for other things in the future. Also replaced a few hardcoded values (hours, seconds etc.) with constants to make the code more consistent.

Edit: tested with normal events, split day events and events with locations.

### CODE REVIEW

**Comment by rejas** [comment]
Nice idea, thwnks. would you feel confident writing a test for this feature?

**Comment by rejas** [comment]
Nice idea, thwnks. would you feel confident writing a test for this feature?

**Review by rejas — CHANGES_REQUESTED** [review]

File: modules/default/calendar/calendar.js [hunk 1]
```diff
@@ -491,6 +496,11 @@ Module.register("calendar", {
 	 * @returns {object[]} Array with events.
 	 */
 	createEventList: function (limitNumberOfEntries) {
+		const oneSecond = 1000; // 1,000 milliseconds
```
[the remark this answers is not in the document]
Comment (retroflex) [reply modules/default/calendar/calendar.js]: Fixed

**Comment by retroflex** [comment]
> Nice idea, thwnks. would you feel confident writing a test for this feature?

@rejas I haven't written tests for MagicMirror or node/js before, so not really. I had a quick check and it seems it uses test ics files: https://github.com/MichMich/MagicMirror/tree/master/tests/configs/data
Not sure how to build an ics file containing events for today and tomorrow, as that would depend on when the test is executed. Any ideas?

**Comment by rejas** [comment]
The trick would be here to change (mock) the date of the system when running the test. Something similar could be either in compliments/weather/calendar tests...

**Comment by retroflex** [comment]
@rejas after upgrading npm & node I got the tests to run with: npm run test:e2e
Is there any way to just run the calendar test?

**Comment by rejas** [comment]
I think by adding the filename of the test behind the jest call in the nom script in package json

**Comment by retroflex** [comment]
This worked fine: `npm run test:e2e -- calendar_spec.js`
Or to run unit tests too: `npm run test -- calendar_spec.js`

However when running it on my RPi3b I get this: 
```Error: Could not load script: "http://localhost:8080/modules/default/calendar/calendar.js```

And after that this: ```
FATAL ERROR: Reached heap limit Allocation failed - JavaScript heap out of memory
Aborted
```

So as much as I like tests, I'm not sure I'll get anywhere with this.

**Review by rejas — APPROVED** [review]

**Comment by rejas** [comment]
Yes, adding tests here is not that easy. I opened a ticket for a general solution to this: https://github.com/MichMich/MagicMirror/issues/2942

And your PR will be merged now, thanks for the work!

**Comment by retroflex** [comment]
Thanks s lot @rejas

**Pull request merged**